packages/apps/fortune/exchange-oracle/server/src/modules/job/job.controller.ts

```diff
@@ -46,12 +45,14 @@ export class JobController {
     );
   }
 
-  @UseGuards(new SignatureAuthGuard([Role.Recording, Role.Reputation]))
-  @Patch('invalid-solution')
-  invalidJobSolution(
+  @UseGuards(
```

Comment:
Web should move this into a webhook controller according to docs. Create a webhook module